CartoDB/cartodb
# Reload map instantiation when provider has changed

Related ticket: https://github.com/CartoDB/cartodb/issues/11910

Fixed problem found above and also #11445. It also has changes in CartoDB.js and Deep-Insights.js, why not??.

## How to test it
- Enable `static-maps` feature flag for your user.
- Enable also google Basemaps for your user 🤗 .
- Use the image-export feature with the current basemap. It should work properly.
- Change it to a CARTO one, for example, and export the image. It should work properly.
- Get back to a google basemap, and check it again :dancer:.
- Also notice that if you change to a `plain` (color) basemap, #11445 is fixed.

cc @javierarce
